purchase_request_to_rfq/models/purchase_order.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+from openerp import _, api, exceptions, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    @api.model
+    def _purchase_request_confirm_message_content(self, po, request,
```

Comment:
Because this method is used to obtain the content of the message. Used  within a loop of the api.multi method _purchase_request_confirm_message